internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/controller/TestAdminToolEndToEnd.java

```diff
@@ -140,6 +145,20 @@ public void testWipeClusterCommand() throws Exception {
       MultiStoreResponse multiStoreResponse = controllerClient.queryStoreList(false);
       Assert.assertEquals(multiStoreResponse.getStores().length, 0);
 
+      // Wait until all topics are indeed removed from Kafka service.
+      PubSubTopicRepository pubSubTopicRepository = new PubSubTopicRepository();
+
+      PubSubTopic testStoreTopic1 = pubSubTopicRepository.getTopic(Version.composeKafkaTopic(testStoreName1, 1));
```

Comment:
why not just do `new PubSubTopicImpl(Version.composeKafkaTopic(testStoreName1, 1))`?

Reply:
Thanks for the review Adam. I think that's simpler, will fix it.

Reply:
In fact, if I go with what you suggested, I have to make `PubSubTopicImpl` constructor public , which seems to disagree the author's intention. maybe I will just keep it as it is.

```
  /**
   * N.B. leaving this constructor package-private, as the intent is not necessarily the instantiate this object
   * from anywhere in the code. Most likely, it would be preferable to have some form of singleton topic-registry where
   * i
```nstances can be gotten from, so that the internal state is computed just once, and then reused for the lifetime
   * of the topic.
   */
  PubSubTopicImpl(String name) {
    this.name = Objects.requireNonNull(name, "Topic name cannot be null");
    this.pubSubTopicType = PubSubTopicType.getPubSubTopicType(name);
    this.storeName = Version.parseStoreFromKafkaTopicName(name);
  }